recipes/poolsnp/build.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/bash
+
+set -ex
+
+BASE_DIR=$(dirname "$0")
+SCRIPTS_DIR="$BASE_DIR/scripts"
+
+if [ ! -d "$SCRIPTS_DIR" ]; then
+    echo "The scripts directory is missing. It should be placed in the same directory as the PoolSNP.sh script."
+    exit 1
+fi
+
+chmod +x PoolSNP.sh
+
+cp PoolSNP.sh $PREFIX/bin
+cp -r $SCRIPTS_DIR $PREFIX/bin
+
+chmod +x $PREFIX/bin/PoolSNP.sh
```

Comment:
This is not really needed. You did it already at line 13 (https://github.com/bioconda/bioconda-recipes/pull/49156/files#diff-3180f716065e95d04e19dba02fcb40838a15acf2f2d2772cf255515c2ed19a48R13)

Reply:
Removed duplicate entries